packages/temen/src/conformsTo/index.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * 'object'가 인라인 함수 'source'와 일치하는지 확인
+ * properties of `source` with the corresponding property values of `object`.
+ *
+ *
+ * @param {Object} object 검사할 오브젝트
+ * @param {Object} source 오브젝트의 프로퍼티를 검증할 오브젝트
+ * @returns {boolean} 'object'가 일치하면 'true'를 반환하고, 그렇지 않으면 'false'를 반환
+ * @example
+ *
+ * const object = { 'a': 1, 'b': 2 }
+ *
+ * conformsTo(object, { 'b': function(n) { return n > 1 } })
+ * // => true
+ *
+ * conformsTo(object, { 'b': function(n) { return n > 2 } })
+ * // => false
+ */
+
+type Conform<T> = {
+  [P in keyof T]: (args: T[keyof T]) => any;
```

Comment:
```suggestion
  [P in keyof T]: (args: T[keyof T]) => unknown;
```
`unknown`으로 변경하는 것이 더 Type Safely할 것 같슴다!

Reply:
근데 이거 각 프로퍼티를 검증하는 함수면 `[P in keyof T]: (args: T[keyof T]) => boolean;`이 되어야 하는 것 아닌가유....?
이 함수 써본 적이 없어서 잘 모르겠네유 🤔

Reply:
흠.. 자바스크립트 유틸을 타입스크립트로 옮기려니 괴리가 있네여,, 강경하게 불리언으로 가야겠읍니다.

Reply:
https://github.com/Lubycon/lubycon-frontend-libraries/pull/109/commits/615bb1876c62d06a298f081639b7fa10e9789b15 수정했슴당